app/utils/newsFeed/newsFeedSync.ts

```diff
@@ -153,6 +153,9 @@ const removeKeysAndSaveNewEntriesAsTransaction = async (
 const aggregatePosts = async ({ from }: { from: Date }): Promise<RedisNewsFeedEntry[]> => {
   // posts fetched from prisma, hence no pagination required
   const posts = await getPostsStartingFrom(dbClient, from);
+  if (!posts || posts.length === 0) {
+    logger.warn('No posts found to sync');
```

Comment:
I would change it rather to `if (posts.length === 0) { logger.**info**(...)}`, because when the posts are aggregated, it could possibly happen that there are no posts to sync (from "syncFrom" timstamp). And add try, catch block to the `getPostsStartingFrom` method to catch the error which can occur when fetching the posts.